int8 robot_id
int8 area_accomplished
---
bool confirmation